blurring_as_a_service/settings/settings_schema.py

```diff
@@ -108,13 +111,20 @@ class DatabaseCredentialsSpec(SettingsSpecModel):
     client_id: str
 
 
-class InferenceCustomerPipelineSpec(SettingsSpecModel):
-    model_name: str
-    model_version: str
-    model_parameters: InferenceModelParameters
-    datastore_input_structured: str
-    datastore_output: str
-    datastore_output_path: str = None
+class InferencePipelineSpec(SettingsSpecModel):
+    model_params: InferenceModelParameters
+    inputs: Dict[str, str] = None
+    outputs: Dict[str, str] = None
+    target_classes: List[int] = None
+    sensitive_classes: List[int] = []
+    target_classes_conf: Optional[float] = None
+    sensitive_classes_conf: Optional[float] = None
+    output_image_size: Optional[Tuple[int, int]] = None
+    save_detection_images: bool = False
+    save_detection_labels: bool = True
+    save_all_images: bool = False
```

Comment:
Perhaps `save_all_images` should be True for BaaS as default, since we also want to save images that do not contain any persons to blur. I know we overwrite this in the yml anyway, but just to avoid mistakes?